chapters/lexicalstructure.tex

```diff
@@ -247,6 +247,24 @@ \subsection{Strings}\label{strings}
 \end{lstlisting}
 \end{nonnormative}
 
+\subsection{Units of Literal Constants}\label{units-literal-constants}
+
+Literal \lstinline!Real! and \lstinline!Integer! constants are implicitly assumed to have a dimensionless unit if they appear in expressions containing variables, parameters, or constants that have a non-empty \lstinline!unit! attribute string, or function calls that have a non-empty \lstinline!unit! attribute of the output variable. The same rule applies to function call inputs.
```

Comment:
```suggestion
The following rules apply:
- The result of `x + L`, `L + x`, `x - L`, `L - x`, `x*L`, `L*x`, `x/L`, where `x` is a variable with non-empty \lstinline!unit! attribute string `<s>` and `L` is a literal constant, shall have the same \lstinline!unit! attribute string of `x`.
- The result of `L/x` shall the the \lstinline!unit! attribute string `1/(<s>)`. 
- The result of `x^L` shall have the \lstinline!unit! attribute string of `product(fill(x, L))` if L is a positive Integer, or the unit of `1/product(fill(x, L))` if L is a negative Integer
- If either side of a relational operator is a literal constant, then it is assumed to have the same \lstinline!unit! attribute string of the other side, if that is well-defined.
```

Reply:
Rewrote first part based on the results of the discussion so far

Reply:
Sorry, I had another look at Henrik examples. The first item in the list says that in the case of `x * L`, "`L` shall have the same unit attribute as `x`", which does mean that:
 ```
Real x(unit = "m") = 1;
Real y(unit = "m2") = x * 10^2; /* OK, due to wildcard effect of 10^2. */
 ```

Reply:
Ha, well spotted! 
The addition should probably get a separate item.

Reply:
Updated the text based on @qlambert-pro's suggestion.